dvc/command/experiments.py

```diff
@@ -145,8 +148,7 @@ def _collect_rows(
             else:
                 name_rev = base_rev
             name = exp.get("name", name_rev)
-            row.append(f"{name}")
-            style = "bold"
+            text = Text(name, style="bold")
```

Comment:
@pmrowla, was this a bug of not using it?

Reply:
no, this style is meant to be applied to the entire row (which is why the original code yielded a tuple of `(row_data, row_style)`

Reply:
@pmrowla, but this wasn't being used at all.

Reply:
ah I see what you meant now. The row style was intended to be used later in the calling function (see the other comment below)